### PR TITLE
Clean up taint_manager_test.go 

### PR DESCRIPTION
**Pull request by muyangren2**
Clean up taint_manager_test.go

#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
image

#### Which issue(s) this PR fixes:

Fixes #
https://github.com/kubernetes/kubernetes/issues/111308
#### Does this PR introduce a user-facing change?

```release-note
"NONE"
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by muyangren2** [comment]
@alculquicondor  I made a wrong commit action, I have resubmitted at here

File: pkg/controller/nodelifecycle/scheduler/taint_manager_test.go [hunk 1]
```diff
@@ -41,10 +41,10 @@ import (
 
 var timeForControllerToProgress = 500 * time.Millisecond
 
-func getPodsAssignedToNode(ctx context.Context, c *fake.Clientset) GetPodsByNodeNameFunc {
+func getPodsAssignedToNode(c *fake.Clientset) GetPodsByNodeNameFunc {
```
Comment (alculquicondor) [inline pkg/controller/nodelifecycle/scheduler/taint_manager_test.go:44]: why remove the context from here? this function needs it?

**Comment by dims** [comment]
If you still need this PR then please rebase, if not, please close the PR

**Comment by alculquicondor** [comment]
ping @muyangren2

**Pull request closed without merging**